Makefile

```diff
@@ -34,6 +69,68 @@ coverage:
 snapshot:
 	forge snapshot
 
+.PHONY: abis
+abis: build
+	@set -e; \
+	mkdir -p "${ABI_PATH}"; \
+	jq '.abi' "${HUB_JSON_PATH}" > "${HUB_ABI_PATH}"; \
+	jq '.abi' "${FORWARDER_JSON_PATH}" > "${FORWARDER_ABI_PATH}"; \
+	jq '.abi' "${TOKEN_JSON_PATH}" > "${TOKEN_ABI_PATH}"
+
+.PHONY: docs
+docs:
+	@forge doc
```

Comment:
why not `forge doc`? Is it too verbose ?

Reply:
The `@` just means that the command itself is not printed by `make`, which I find redundant here. Any output from the command will still be shown.